src/pal/src/locale/utf8.cpp

```diff
@@ -1399,7 +1399,6 @@ class UTF8Encoding
 
         EncodeChar:
 
-#ifdef FASTLOOP
```

Comment:
It is because it has `#define FASTLOOP` at the top. The tool sometimes applies such defines. I thought I had reverted them all. Some future pass could look at such things, and `#if (0)`, and decide whether to remove any. Maybe they are debugging code.
Thanks for catching.